pallets/kensetsu/src/migrations.rs

```diff
@@ -97,3 +97,66 @@ pub mod init {
         }
     }
 }
+
+/// Due to bug in stability fee update some extra KUSD were minted, this migration burns and sets
+/// correct amounts.
+pub mod stage_correction {
+    use crate::{CDPDepository, CollateralInfos, Config};
+    use assets::AssetIdOf;
+    use common::AssetInfoProvider;
+    use common::Balance;
+    use frame_support::dispatch::Weight;
+    use frame_support::traits::OnRuntimeUpgrade;
+    use sp_arithmetic::traits::Zero;
+    use sp_core::Get;
+    use std::collections::BTreeMap;
+    use std::marker::PhantomData;
+
+    pub struct CorrectKusdBalances<T>(PhantomData<T>);
+
+    impl<T: Config + permissions::Config + technical::Config> OnRuntimeUpgrade
+        for CorrectKusdBalances<T>
+    {
+        fn on_runtime_upgrade() -> Weight {
+            let mut weight = Weight::zero();
+
+            let mut collateral_debts: BTreeMap<AssetIdOf<T>, Balance> = BTreeMap::new();
+            let mut total_debt = Balance::zero();
+
+            for (_, cdp) in CDPDepository::<T>::iter() {
+                *collateral_debts.entry(cdp.collateral_asset_id).or_default() += cdp.debt;
+                total_debt += cdp.debt;
+                weight += <T as frame_system::Config>::DbWeight::get().reads(1);
+            }
+
+            for asset_id in CollateralInfos::<T>::iter_keys() {
+                CollateralInfos::<T>::mutate(asset_id, |collateral_info| {
+                    let collateral_info = collateral_info.as_mut().unwrap();
```

Comment:
Please, avoid using `unwrap`s in migrations

Reply:
`asset_id` is from iter, entry must exists.